Add feature for splicing READMEs into the documentation (#11519)
PrairieLearn/PrairieLearn: contrib/question_converters/README.md

Author: Now resolved.

> # Question converters
> 
> ## Convert Learning Management System (LMS) questions to PrairieLearn questions

Reviewer: Nit, workshopping a bit:

```suggestion
# LMS question converters
```

> Within this directory are a set of conversion tools for moving previously written questions
> in a different Learning Management System (LMS) to the question format for PrairieLearn.

Reviewer: Nit, a little more workshopping:

```suggestion
This directory contains tools for converting questions from another Learning Management Systems (LMS) to PrairieLearn's question format.
```
